<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629693102825' stop='1629693308237'>
    <name>verify basic project append funtion</name>
    <title>verify basic project append funtion</title>
    <test-cases>
        <test-case start='1629693102826' status='failed' stop='1629693104386'>
            <name>Qick append verify add new line</name>
            <title>Qick append verify add new line</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='4c14d76d-fa13-4f23-a96c-62a28143ef82-attachment.png' type='image/png' size='59232'/>
            </attachments>
            <failure>
                <message>Failed: element click intercepted: Element &lt;a _ngcontent-oyg-c1="" class="nav-link active" href="/projects" routerlinkactive="active">...&lt;/a> is not clickable at point (720, 30). Other element would receive the click: &lt;div _ngcontent-oyg-c13="" class="modal">...&lt;/div>
                  (Session info: chrome=92.0.4515.159)
                  (Driver info: chromedriver=92.0.4515.107 (87a818b10553a07434ea9e2b6dccf3cbe7895134-refs/branch-heads/4515@{#1634}),platform=Windows NT 10.0.19043 x86_64)</message>
                <stack-trace>WebDriverError: element click intercepted: Element &lt;a _ngcontent-oyg-c1="" class="nav-link active" href="/projects" routerlinkactive="active">...&lt;/a> is not clickable at point (720, 30). Other element would receive the click: &lt;div _ngcontent-oyg-c13="" class="modal">...&lt;/div>
                  (Session info: chrome=92.0.4515.159)
                  (Driver info: chromedriver=92.0.4515.107 (87a818b10553a07434ea9e2b6dccf3cbe7895134-refs/branch-heads/4515@{#1634}),platform=Windows NT 10.0.19043 x86_64)
                    at Object.checkLegacyResponse (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\error.js:546:15)
                    at parseHttpResponse (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\http.js:509:13)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\http.js:441:30
                    at runMicrotasks (&lt;anonymous>)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: WebElement.click()
                    at Driver.schedule (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at WebElement.schedule_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
                    at WebElement.click (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:2092:17)
                    at actionFn (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:461:65
                    at ManagedPromise.invokeCallback_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2927:27Error: 
                    at ElementArrayFinder.applyAction_ (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as click] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as click] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:831:22)
                    at Function.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\utils\function-util.ts:127:15)
                    at Generator.next (&lt;anonymous>)
                    at fulfilled (E:\vmware\Loop\GitHub\e2e\node_modules\tslib\tslib.js:112:62)
                    at runMicrotasks (&lt;anonymous>)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: Run it("Qick append verify add new line") in control flow
                    at UserContext.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:94:19)
                    at attempt (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:64:48
                    at ControlFlow.emit (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\events.js:62:21)
                    at ControlFlow.shutdown_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2674:10)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2599:53
                From asynchronous test: 
                Error: 
                    at Suite.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\append-project-spec.ts:14:3)
                    at addSpecsToSuite (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\append-project-spec.ts:10:1)
                    at Module._compile (node:internal/modules/cjs/loader:1101:14)
                    at Module.m._compile (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:422:23)
                    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
                    at Object.require.extensions.&lt;computed> [as .ts] (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:425:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629693104387' status='failed' stop='1629693305553'>
            <name>Qick append verify delete new line</name>
            <title>Qick append verify delete new line</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='9c465db6-8479-4de8-b2f4-64a234b04edf-attachment.png' type='image/png' size='59232'/>
                <attachment title='Screenshot' source='615f688a-b32a-4cea-8df0-336801c5028d-attachment.png' type='image/png' size='59232'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (node:internal/timers:557:17)
                    at processTimers (node:internal/timers:500:7)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629693305554' status='failed' stop='1629693306899'>
            <name>Qick append verify publish</name>
            <title>Qick append verify publish</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='d2396a89-a878-452e-a00f-9b702f0916fd-attachment.png' type='image/png' size='59232'/>
            </attachments>
            <failure>
                <message>Failed: Index out of bound. Trying to access element at index: -1, but there are only 0 elements that match locator By(css selector, clr-dg-row)</message>
                <stack-trace>NoSuchElementError: Index out of bound. Trying to access element at index: -1, but there are only 0 elements that match locator By(css selector, clr-dg-row)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:274:27
                    at ManagedPromise.invokeCallback_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at runMicrotasks (&lt;anonymous>)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: Run it("Qick append verify publish") in control flow
                    at UserContext.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:94:19)
                    at attempt (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:64:48
                    at ControlFlow.emit (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\events.js:62:21)
                    at ControlFlow.shutdown_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2674:10)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2599:53
                From asynchronous test: 
                Error: 
                    at Suite.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\append-project-spec.ts:23:3)
                    at addSpecsToSuite (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\append-project-spec.ts:10:1)
                    at Module._compile (node:internal/modules/cjs/loader:1101:14)
                    at Module.m._compile (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:422:23)
                    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
                    at Object.require.extensions.&lt;computed> [as .ts] (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:425:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629693306900' status='failed' stop='1629693308236'>
            <name>File append verify selete existing file</name>
            <title>File append verify selete existing file</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='e9166196-b13c-4de9-89dc-739389b52fc6-attachment.png' type='image/png' size='59232'/>
            </attachments>
            <failure>
                <message>Failed: element click intercepted: Element &lt;a _ngcontent-oyg-c1="" class="nav-link active" href="/projects" routerlinkactive="active">...&lt;/a> is not clickable at point (720, 30). Other element would receive the click: &lt;div _ngcontent-oyg-c13="" class="modal">...&lt;/div>
                  (Session info: chrome=92.0.4515.159)
                  (Driver info: chromedriver=92.0.4515.107 (87a818b10553a07434ea9e2b6dccf3cbe7895134-refs/branch-heads/4515@{#1634}),platform=Windows NT 10.0.19043 x86_64)</message>
                <stack-trace>WebDriverError: element click intercepted: Element &lt;a _ngcontent-oyg-c1="" class="nav-link active" href="/projects" routerlinkactive="active">...&lt;/a> is not clickable at point (720, 30). Other element would receive the click: &lt;div _ngcontent-oyg-c13="" class="modal">...&lt;/div>
                  (Session info: chrome=92.0.4515.159)
                  (Driver info: chromedriver=92.0.4515.107 (87a818b10553a07434ea9e2b6dccf3cbe7895134-refs/branch-heads/4515@{#1634}),platform=Windows NT 10.0.19043 x86_64)
                    at Object.checkLegacyResponse (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\error.js:546:15)
                    at parseHttpResponse (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\http.js:509:13)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\http.js:441:30
                    at runMicrotasks (&lt;anonymous>)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: WebElement.click()
                    at Driver.schedule (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at WebElement.schedule_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
                    at WebElement.click (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:2092:17)
                    at actionFn (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:461:65
                    at ManagedPromise.invokeCallback_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2927:27Error: 
                    at ElementArrayFinder.applyAction_ (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as click] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as click] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:831:22)
                    at Function.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\utils\function-util.ts:127:15)
                    at Generator.next (&lt;anonymous>)
                    at fulfilled (E:\vmware\Loop\GitHub\e2e\node_modules\tslib\tslib.js:112:62)
                    at runMicrotasks (&lt;anonymous>)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: Run it("File append verify selete existing file") in control flow
                    at UserContext.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:94:19)
                    at attempt (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:64:48
                    at ControlFlow.emit (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\events.js:62:21)
                    at ControlFlow.shutdown_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2674:10)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2599:53
                From asynchronous test: 
                Error: 
                    at Suite.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\append-project-spec.ts:27:3)
                    at addSpecsToSuite (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\append-project-spec.ts:10:1)
                    at Module._compile (node:internal/modules/cjs/loader:1101:14)
                    at Module.m._compile (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:422:23)
                    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
                    at Object.require.extensions.&lt;computed> [as .ts] (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:425:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>